NBXplorer/Properties/launchSettings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:4771/",
+      "sslPort": 0
+    }
+  },
+  
+  "profiles": {
+    "NBXplorer - BTC on Mainnet": {
+      "commandName": "Project",
+      "commandLineArgs": "--btcrpcuser=myuser --btcrpcpassword=mypassword",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:4774",
+      "launchUrl": "v1/cryptos/btc/status"
+    },
+
+    "NBXplorer - BTC+LTC on Regtest": {
+      "commandName": "Project",
+      "commandLineArgs": "--chains=btc,ltc --network=regtest --btcrpcuser=myuser --btcrpcpassword=mypassword --ltcrpcuser=myuser --ltcrpcpassword=mypassword",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:4774",
+      "launchUrl": "v1/cryptos/ltc/status"
+    },
+
+    "NBXplorer - DASH on Testnet": {
+      "commandName": "Project",
+      "commandLineArgs": "--chains=dash --network=test --dashrpcuser=myuser --dashrpcpassword=mypassword",
```

Comment:
Same remark, remove explicit rpc settings

Reply:
I have vague memory that dash don't support cookiefile authentication for god know why.

Reply:
It is alright then to leave it as it is, demonstrating that RPC username/password is a valid authentication method as well?

Reply:
for dash we can't do anything about it but to remove as is

Reply:
`dashd` supports cookie authentication as well.

This is from its debug.log:
> 2019-09-03 08:39:34 HTTP: creating work queue of depth 16
> 2019-09-03 08:39:34 No rpcpassword set - using random cookie authentication
> 2019-09-03 08:39:34 Generated RPC authentication cookie C:\Users\\....\AppData\Roaming\DashCore\testnet3\.cookie
> 2019-09-03 08:39:34 HTTP: starting 4 worker threads

And this is the output of NBXplorer when running with the `--chains=dash --network=test` parameters:
![image](https://user-images.githubusercontent.com/910321/64157593-405ef980-ce37-11e9-8f5e-381a4bdd940e.png)

Reply:
Ok then you should remove dashrpcuser

Reply:
Ok, removed.